File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int uniqueELementIndex = 0;
        int n = nums.size();
        for(int i=1; i<n; i++){
            if(nums[i] != nums[uniqueELementIndex]){
                uniqueELementIndex++;
                nums[uniqueELementIndex] = nums[i];
            }
        }
        return uniqueELementIndex+1;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int len = nums.length;
        int uniqueElementIndex = 0;
        for(int i = 1; i<len; i++){
            if(nums[i] != nums[uniqueElementIndex]){
                uniqueElementIndex++;
                nums[uniqueElementIndex] = nums[i];
            }
        }
        return uniqueElementIndex+1;
    }
}